src/pyseekdb/client/client_base.py

```
@@ -1223,6 +1250,83 @@ def get_or_create_collection(
+        return collection_id
+
+    def _collection_fork(self, collection: Collection, forked_name: str) -> None:
+        """
+        Fork a collection
+
+        Args:
+            collection: Collection to fork
+            forked_name: Forked collection name
+        """
+        if not self._fork_enabled():
+            raise ValueError("Fork is not enabled for this database")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)